src/_pytest/pytester.py

```diff
@@ -370,9 +370,13 @@ def __init__(self, ret, outlines, errlines, duration):
         self.duration = duration
 
     def __repr__(self):
+        try:
+            ret = pytest.ExitCode(self.ret)
```

Comment:
this should  convert in the ctor instead

Reply:
* try to

Reply:
Done, also changed from:

```
<RunResult ret=<ExitCode.TESTS_FAILED: 1> len(...
``` 

To:

```
<RunResult ret=ExitCode.TESTS_FAILED len(...
```

Because I believe it reads better.